devinova/app/components/layout/TextWithButton.tsx

```diff
@@ -0,0 +1,27 @@
+import BodyThreeHeadings from "../ui/BodyThreeHeadings";
+import BodyTwoHeadings from "../ui/BodyTwoHeadings";
+import ArrowInCircleBtn from "./buttons/ArrowInCircleBtn";
+import ArrowBtn from "./buttons/ArrowBtn";
+import OpenNewWindowBtn from "./buttons/OpenNewWindowBtn";
+const TextWithButton = ({ data }: { data: any }) => {
+  return (
+    <div>
+      {data.subHeadAbove ? (
+        <BodyThreeHeadings data={data} />
+      ) : (
+        <BodyTwoHeadings data={data} />
+      )}
+      {data.buttonType == "arroCircle" ? (
```

Comment:
Maybe easier to read if it is a case statment?

Reply:
can you give me an example? do you meen is statement?